hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/TransactionManager.java

```diff
@@ -35,44 +35,39 @@
 public class TransactionManager implements Serializable {
 
   private static final Logger LOG = LogManager.getLogger(TransactionManager.class);
-
   private final LockManager lockManager;
+  private final boolean supportsOptimisticConcurrency;
   private Option<HoodieInstant> currentTxnOwnerInstant;
   private Option<HoodieInstant> lastCompletedTxnOwnerInstant;
-  private boolean supportsOptimisticConcurrency;
 
   public TransactionManager(HoodieWriteConfig config, FileSystem fs) {
     this.lockManager = new LockManager(config, fs);
     this.supportsOptimisticConcurrency = config.getWriteConcurrencyMode().supportsOptimisticConcurrencyControl();
   }
 
-  public synchronized void beginTransaction() {
+  public void beginTransaction() {
     if (supportsOptimisticConcurrency) {
-      LOG.info("Transaction starting without a transaction owner");
```

Comment:
Can we have the first info logging. Will help us debug issues where a writer is waiting to acquire a lock. 
same for next method too.

Reply:
added back.